### DISCUSSION TITLE
Get a list of all Bootstrap variables in JSON

### DISCUSSION THREAD

**Discussion by nicolasbadia** [twbs/bootstrap#35742] [Q&A]
Is there an easy way to get all the variables in JSON format grouped by categories. Something like this:

```
{
  "Colors": [
    {
      "key": "$blue",
      "value": "#007bff",
      "type": "color"
    },
    ...
  ],
  "Core": [
    {
      "key": "$enable-caret",
      "value": "true",
      "type": "boolean"
    },
    ....
  ]
}
```

**Comment by GeoSot** [comment]
You could try a library/package like this  https://github.com/niksy/get-sass-vars. But I don't think you will get to the point to recognize the variable type :smiley:
